client/src/pages/GamePage.tsx

```diff
@@ -91,29 +93,27 @@ const GamePage = observer(() => {
     };    
 
     const winner = data.sort((a, b) => b.text - a.text)[0]?.userName;
-        
-    const isStartedStyle = styles[gameIsStarted ? 'startGame' : 'startGameHidden']
 
     return (
-        <div className={styles.gameWrapper}>
-            <div style={{display: 'flex'}}>
-                <button
+        <PageWrapper>
+            <div style={{display: 'flex', gap: '20px'}}>
+                <Button
                     title="COPY"
-                    className={styles.buttonEnterName}
                     onClick={() => copy()}
```

Comment:
Also, it will be good to rename it with handleClipboardCopy, for example.
```suggestion
                    onClick={copy}
```